addons/account_payment/models/account_move.py

```diff
@@ -51,7 +56,8 @@ def _has_to_be_paid(self):
                 self.amount_residual
                 # FIXME someplace we check amount_residual and some other amount_paid < amount_total
                 # what is the correct heuristic to check ?
-                or not transactions
+                or not (transactions or pending_transactions)
+                # consider pending txs or not ?
```

Comment:
I'd say yes, but we can keep the current behavior for now if it wasn't the case and rethink all portals as a whole in master.

Reply:
Well, that's the problem, it was somewhat considered before (see the pending_manual_txs thing in the template before).
Let's consider them for now, worst case we can change our minds if needed.